Firmware/language.h

```diff
@@ -20,14 +21,67 @@
 #define STRINGIFY_(n) #n
 #define STRINGIFY(n) STRINGIFY_(n)
 
+//section progmem0 will be used for localized translated strings
+#define PROGMEM_I2 __attribute__((section(".progmem0")))
+//section progmem1 will be used for localized strings in english
+#define PROGMEM_I1 __attribute__((section(".progmem1")))
+//section progmem2 will be used for not localized strings in english
+#define PROGMEM_N1 __attribute__((section(".progmem2")))
+
+#if (LANG_MODE == 0) //primary language only
+#define _I(s) (__extension__({static const char __c[] PROGMEM_I1 = s; &__c[0];}))
+#define ISTR(s) s
+#define _i(s) _I(s)
+#define _T(s) s
+#else //(LANG_MODE == 0)
+#define _I(s) (__extension__({static const char __c[] PROGMEM_I1 = "\xff\xff"s; &__c[0];}))
```

Comment:
Hi, try to compile the firmware but get allways:

`language.h:37: error: unable to find string literal operator 'operator"" s'`

and
```
exit status 1
unable to find string literal operator 'operator"" s'
```

What do i miss? Using Arduino IDe 1.6.8  portable with Arduino AVR Boards 1.6.11 (because later updates 1.6.12-1.6.21 give more errors) as the Arduino IDE 1.8.5